student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   def initialize(age, classroom, parents_permission: true, name: 'Unknown')
     super(age, parents_permission: parents_permission, name: name)
     @classroom = classroom
+    classroom.add_student(self)
   end
 
   def play_hooky
```

Comment:
- [ ] Kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom). Like so:
```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
I completely missed that, thanks for pointing it out! @ShadyShawkat :smile: